governance/votes/Executive vote - April 18, 2024.md

```diff
@@ -0,0 +1,128 @@
+
+### Whitelisting New Address in the RWA015-A Output Conduit
+
+- **Authorization:** [Atlas Section 2.5.10](https://mips.makerdao.com/mips/details/MIP101#2-5-10-avc-member-participation-rewards)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-04-18/24084)
+
+If this executive proposal passes, following the recommendation of the Governance Facilitators in collaboration with the spell team, the following operations will be performed:
+
+- Raise the PSM_PAX_A Debt Ceiling to **100,000 DAI**.
+- Call `push(uint256 amt)` on the the PSM_PAX_A_INPUT_CONDUIT_JAR contract with an amount of **84,211.27 USDP**, depositing the contents of the contract to the PSM and receiving an identical amount of Dai into the PSM_PAX_A_JAR contract.
+- Call `void()` on the PSM_PAX_A_JAR contract to deposit the Dai to the Surplus Buffer.
+- Set PSM-PAX-A Debt Ceiling to **0 DAI**.
+- Decrease Debt Ceiling of PSM_PAX_A back to **0 DAI**.
+
+This will have the following net results:
+
+- The PSM_PAX_A Debt Debt Ceiling remains at 0 DAI after the transaction has been executed.
+- Dai generated from PSM_PAX_A increases by **84,211.27 DAI**.
+- USDP balance of MCD_JOIN_PSM_PAX_A increases by **84,211.27 USDP**.
+- USDP balance of PSM_PAX_A_INPUT_CONDUIT_JAR decreases to **0 USDP**.
+- Surplus Buffer increases by **84,211.27 DAI**.
+
+### Triggering Spark Proxy Spell
+
+- **Authorization:** [Ecosystem Approval](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033/7), Governance Polls [1095](https://vote.makerdao.com/polling/QmfGV2vt), [1096](https://vote.makerdao.com/polling/QmSYZSCQ), [1097](https://vote.makerdao.com/polling/QmcRdMyA), [1098](https://vote.makerdao.com/polling/QmSh8gyC), [1099](https://vote.makerdao.com/polling/QmUhT32b), [1100](https://vote.makerdao.com/polling/QmZND8WW), [1104](https://vote.makerdao.com/polling/QmVXriiT), [1106](https://vote.makerdao.com/polling/QmVsKsGa)
+- **Proposal:** [Forum post "[Apr 4, 2024] Proposed Changes to SparkLend for Upcoming Spell"](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033)
+
+If this executive proposal passes, the SparkLend Proxy Spell on Ethereum and Gnosis at [0x3d1DD14Fa08163E7f64b0abf0F514f6276f50882](https://etherscan.io/address/0x3d1DD14Fa08163E7f64b0abf0F514f6276f50882) will be executed, containing the following items:
+
+- [Ethereum+Gnosis] Upgrade Pool Implementation to Disable Flashloan into Borrow Feature
+- [Gnosis] Onboard sxDAI
+- [Gnosis] Onboard EURe
+- [Gnosis] Onboard USDC
+- [Gnosis] Onboard USDT
+- [Gnosis] Refresh Existing Market Parameters
+- [Gnosis] Update xDAI Oracle Parameter
+- [Ethereum] Activate the SparkLend External Security Access Multisig
```

Comment:
:red_circle: / :yellow_circle: **moderate/severe**

We support itemizing inclusions for Spark Proxy spells as voters still need to understand the details of their votes.

See: https://github.com/makerdao/community/blob/master/governance/votes/Executive%20Vote%20-%20April%204%2C%202024.md#trigger-sparklend-proxy-spell